demisto_sdk/commands/split/tests/ymlsplitter_test.py

```diff
@@ -29,6 +30,105 @@ def test_extract_long_description(tmpdir):
     os.remove(extractor.output)
 
 
+def test_extract_modeling_rules(tmpdir):
+    """
+    Given:
+        - A unified YML of a Modeling Rule
+    When:
+        - run extract_rules
+    Then:
+        - Ensure that the rules were extracted
+    """
+    extractor = YmlSplitter(input=f'{git_path()}/demisto_sdk/tests/test_files/modelingrule-OktaModelingRules.yml',
+                            output=str(tmpdir.join('temp_rules.xif')), file_type='modelingrule')
```

Comment:
You don't really need the output here as you are not using it in the `extract_rules` function, you can just pass it straight to the function.
```suggestion
```

Reply:
This is true for all the tests you are doing for both `extract_rules` and `extract_rule_schema`.

Reply:
What's the difference? if I pass it as part of the YMLSplitter or directly to the function

Reply:
Cause it's confusing - when you do `YmlSplitter(output='some_path')` it makes me think this is being used inside the flow in some way where in reality both `extract_rules` and `extract_rule_schema` are getting it as an argument.